Framework/API/inc/MantidAPI/Projection.h

```diff
@@ -8,7 +8,7 @@
 
 #include <boost/shared_ptr.hpp>
 
-using namespace Mantid::Kernel;
+// using namespace Mantid::Kernel;
```

Comment:
Thank you for catching and fixing this! :+1: 

Would you please delete the commented out line.